src/main/java/uk/gov/hmcts/divorce/ccd/NoFaultDivorce.java

```diff
@@ -20,7 +20,7 @@
 @Component
 public class NoFaultDivorce implements CCDConfig<CaseData, State, UserRole> {
 
-    public static final String CASE_TYPE = "NO_FAULT_DIVORCE14";
+    public static final String CASE_TYPE = "NO_FAULT_DIVORCE15";
```

Comment:
don't think we need to bump this

Reply:
Ah right, I must be confused. When do we need to bump it? I thought it was when we created or updated the fields

Reply:
if you change the field type then you will need to update